pages/learn/getting-started.md

```diff
@@ -74,13 +74,9 @@ Once you have finished your image configuration, click the *Download {{ $names.o
 
 Now that we have a device or two connected to a {{ $names.company.lower }} application, let's deploy some code.
 
-The recommended way to deploy code is to install the [{{ $names.cli.lower }}][cli]. The easiest way to do this is to use the installer for your OS available on the [releases page][releases]. Choose the latest release of the installer for your OS, and follow the [installation instructions][install].
-
-<img alt="{{ $names.company.lower }} CLI installer" src="/img/common/cli/cli-release.png" width="100%">
-
-__Note:__ You may also install the {{ $names.company.lower }} CLI via npm on a system running NodeJS, as explained in [NPM Installation][npminstall].
-
-To use the [{{ $names.cli.lower }}][cli], you need to login to your {{ $names.company.lower }} account. Login via the terminal using the `{{ $names.company.lower }} login` command:
+The recommended way to deploy code is using {{ $names.cli.lower }}. Follow the [balenaCli
+installation instructions][cli-install]. After it is installed, login to your {{ $names.company.lower}}
+account with the `{{ $names.company.lower }} login` command on the terminal:
```

Comment:
This is the main change that motivated this PR, as [discussed in Flowdock](https://www.flowdock.com/app/rulemotion/r-product/threads/u-0Zurwif8-D8R_ghYbfpAghgF6).

Note that this PR depends on balena-io/balena-cli/pull/2039, which simplifies the installation instructions in the balenaCLI repo, focusing on a single installation option for each operating system (while having a link for "advanced installation options" for users who are interested). The new instructions also contain a direct link to the latest release / version on GitHub.

The screenshot of balenaCLI's GitHub releases page was clickable and seemed to be more confusing than helpful, hence removed. Also removed was the note about the option of installing the CLI via NPM, which is certainly not a good idea for the getting started guide (too complex and not really recommended for most users).